prettypandas/formatters.py

```diff
@@ -52,22 +58,23 @@ def as_unit(v, unit, precision=2, location='suffix'):
         raise TypeError("Precision must be an integer.")
 
     if location == 'prefix':
-        formatter = partial(format_number, prefix=unit)
+        formatter = partial(format_number, prefix=unit, **kwargs)
     elif location == 'suffix':
-        formatter = partial(format_number, suffix=unit)
+        formatter = partial(format_number, suffix=unit, **kwargs)
     else:
         raise ValueError("location must be either 'prefix' or 'suffix'.")
 
     return formatter(v, "0.{}f".format(precision))
 
+#disable use of babel for percentages so we can control precision
+#as_percent = partial(numbers.format_percent,
+#                     locale=LOCALE_OBJ)
+#"""Format number as percentage."""
```

Comment:
Choice of percentage formatter sounds like a good idea.  

I guess a user is likely to want to use either Babel or the custom formatting code all the time for percentages - would a program level default be ok to implement that, and allow overrides on a per call basis to as_percent?